images/linux/scripts/installers/java-tools.sh

```diff
@@ -10,10 +10,13 @@ source $HELPER_SCRIPTS/os.sh
 
 set -e
 
-# Install the Azul Systems Zulu JDKs
-# See https://www.azul.com/downloads/azure-only/zulu/
+# Install GPG Key for Azul Open JDK. See https://www.azul.com/downloads/azure-only/zulu/
 apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 0xB1998361219BD9C9
 apt-add-repository "deb http://repos.azul.com/azure-only/zulu/apt stable main"
```

Comment:
Why do we need Azul repo?

Reply:
There is no Java 7 package in Adopt Open JDK repository:

testAdm2@v-danurgAdopt:/usr/lib/jvm$ sudo apt-get -y install adoptopenjdk-7-hotspot
Reading package lists... Done
Building dependency tree       
Reading state information... Done
E: Unable to locate package adoptopenjdk-7-hotspot

Reply:
Why do we need Azul repo for Ubuntu 20?

Reply:
Thanks for clarifying, I have changed this.